cas-server-support-oidc/src/main/java/org/jasig/cas/services/OidcRegisteredService.java

```diff
@@ -0,0 +1,88 @@
+package org.jasig.cas.services;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.jasig.cas.support.oauth.services.OAuthRegisteredService;
+
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+
+/**
+ * This is {@link OidcRegisteredService}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+@Entity
+@DiscriminatorValue("oidc")
+public class OidcRegisteredService extends OAuthRegisteredService {
+
+    private static final long serialVersionUID = 1310899699465091444L;
+
+    private String jwks;
+    private boolean signIdToken;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['VARIABLE_DEF' should be separated from previous statement.](https://www.codacy.com/app/mmoayyed/cas/file/2503633508/issues/source?bid=3256115&fileBranchId=3273836#l24)